web/themes/custom/bglobal/source/_patterns/01-molecules/hero/_hero.scss

```diff
@@ -0,0 +1,7 @@
+.hero__image__container {
```

Comment:
Will there be any override issues with this hero if main branch has the new hero component in it?

Reply:
Yup, this has been replaced with the code that you wrote and I merged.